CMakeLists.txt

```diff
@@ -355,8 +364,11 @@ if(GCC OR CLANG)
   # Note clang-cl is odd and sets both CLANG and MSVC. We base our configuration
   # primarily on our normal Clang one.
   if (NOT CMAKE_C_STANDARD)
+    message(STATUS "Adding CMAKE_C_FLAGS -std=c99")
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99")
```

Comment:
Do we want to try and compile with c11 here too?

Reply:
This is kind of a weird fallback branch that should only happen for cmake versions between 3.0 and 3.1 which don't support CMAKE_C_STANDARD.